plugin_tests/color_conversion_test.py

```diff
@@ -31,6 +31,7 @@
 # boiler plate to start and stop the server if needed
 def setUpModule():
     base.enabledPlugins.append('HistomicsTK')
+    base.enabledPlugins.append('large_image')
```

Comment:
This shouldn't be necessary, as the HistomicsTK plugin automatically enables the large_image plugin as a dependency.